components/content_settings/core/browser/brave_cookie_settings.h

```diff
@@ -27,9 +29,21 @@ class BraveCookieSettings : public CookieSettings {
                         const GURL& tab_url,
                         content_settings::SettingSource* source,
                         ContentSetting* cookie_setting) const;
+
+  bool IsCookieAccessAllowed(const GURL& url,
```

Comment:
This is kind of confusing now. If I am the user of this class, which method should I choose?
I suggest to at least change names somehow and write comments.

Reply:
Resolved, renamed the functions for clarity.